manage/sawtooth_manage/docker.py

```diff
@@ -135,10 +135,10 @@ def start(self, node_config):
         if node_config.genesis:
             command = 'bash -c "sawtooth admin keygen && \
             sawtooth admin genesis && \
-            validator {} -v"'
+            validator {} -v --public-uri tcp://0.0.0.0:8800"'
```

Comment:
In subprocess, all the validators are running locally on the same machine, so subprocess should be OK, but take a look.